src/CoreBundle/Resources/config/services.xml

```diff
@@ -789,6 +790,12 @@
             <argument type="service" id="request_stack"/>
         </service>
 
+        <service id="chameleon_system_core.resource_collector" class="TCMSResourceCollection">
```

Comment:
The class is not stateless, so we should set the service to non-shared. One could argue that the state is overwritten before it is used, but we should be conservative here.

Reply:
ok